addons/web/static/src/legacy/js/views/view_utils.js

```diff
@@ -21,10 +21,12 @@ var viewUtils = {
      * field for the records in that group.
      *
      * @param {Object} group dataPoint of type list, corresponding to a group
-     * @param {string} groupByField the name of the groupBy field
+     * @param {string} groupByValue the value of the groupby, i.e.
+     *                              field_name:granularity for date/datetime
      * @returns {string | integer | false}
      */
-    getGroupValue: function (group, groupByField) {
+    getGroupValue: function (group, groupByValue) {
```

Comment:
groupByValue should be groupedBy as in getGroupByField I think. Use groupByValue is quite misleading in my opinion.